ssl/statem/statem_clnt.c

```diff
@@ -1347,7 +1347,10 @@ static int set_client_ciphersuite(SSL *s, const unsigned char *cipherchars)
     if (s->hit && (s->session->cipher_id != c->id)) {
         if (SSL_IS_TLS13(s)) {
             const EVP_MD *md = ssl_md(s->ctx, c->algorithm2);
-
```

Comment:
Please retain this blank line between the declaration and the body of the code.

Reply:
done.